lib/apiconfig/apiconfig.go

```diff
@@ -48,8 +48,6 @@ type CalicoAPIConfigSpec struct {
 }
```

Comment:
I suppose we should get rid of the EtcdV2 datastore type now too.

Reply:
yup good point.